docs/tutorial/understanding_ALF_via_the_minimal_working_example.rst

```diff
@@ -401,23 +428,6 @@ In summary, different components on a ALF pipeline are connected by using
 :class:`.TensorSpec` to specify their I/O specs. This also happens within a component,
 for example, between child algorithms, between networks, etc.
 
-More than training pipelines
-----------------------------
-
-Another major effort of ALF is providing an extensive set of high-quality tools
-for RL research, including various algorithms, networks, layers, and environments.
-
-.. code-block:: bash
-
-    alf/algorithms/
-    alf/networks/
-    alf/layers.py
-    alf/environments/
-
-A user can easily experiment them via the conf file. In the example conf,
-:class:`.ActorCriticAlgorithm`, :class:`.ActorDistributionNetwork`, and
-:class:`.NormalProjectionNetwork` are representatives.
-
 Summary
 -------
 
```

Comment:
Minor typo in the first sentence of the last paragraph: It might still be unclear to a user

Reply:
updated